application/controllers/handler.js

```diff
@@ -199,19 +200,23 @@ module.exports = {
 
     return userCtrl.read(util.parseStringToInteger(req.params.id))
       .then((user) => {
-        //console.log('getUser: got user:', user);
+        console.log('getUser: got user:', user);
         if (user !== undefined && user !== null && user.username !== undefined) {
           if (user.deactivated === true) {
             return res(boom.locked('This user is deactivated.'));
           }
 
-          //get groups of a user
           return usergroupCtrl.readGroupsOfUser(req.params.id)
-            .then((array) => {
-              user.groups = array;
+            .then((groupArray) => {
+              user.groups = groupArray;
 
-              return res(prepareDetailedUserData(user)).header(config.JWT.HEADER, jwt.createToken(user));
-            });
+              return userltiCtrl.readLTIsOfUser(req.params.id)
```

Comment:
Please make sure, that in the reject case of userltiCtrl.readLTIsOfUser the service still returns the user.